src/providers/UserActivityProvider/docs/UserActivityProvider.stories.mdx

```diff
@@ -0,0 +1,44 @@
+<Meta title="SDK Providers/UserActivityProvider" />
+
+# UserActivityProvider
+
+The `UserActivityProvider` provides the state of user activity.
```

Comment:
I think it would be good to explain how we determine a user is active, and what potentially use case it fits for. e.g. hide the control bar based on user activity.

Reply:
I agree with @ziyiz-amzn. 'User activity' is a fairly ambiguous term, it would be helpful to clarify what kind of activity in particular this provider is measuring (eg; focus events and mouse movement).

Reply:
Got it, updated the description.